examples/Dojo/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8" />
+    <title>OnLoad Example</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+
+  <body>
+    <h1 id="greeting">Hello</h1>
+    <!-- load Dojo -->
+    <script src="//ajax.googleapis.com/ajax/libs/dojo/1.10.4/dojo/dojo.js"
+            data-dojo-config="async: true"></script>
+
+
+    <p>This is basic example, executing script ater various events.</p>
+    <p>Please check the Dev Console's network tab to inspect the events.</p>
+
+    <hr/>
+
+    <br /> <br />
+
+    <h2>Dojo On Load Service</h2>
+    <p>A event will fire when the application first loads.</p>
+
+    <hr/>
+
+    <br /> <br />
+
+    <h2>Dojo State Service</h2>
+    <p>A event will fire every time the <strong>dojo/router</strong> module fires.  Click one of the buttons below:</p>
+    <button type="button" id="rci-router-trigger-1">Router 1</button>
+    <button type="button" id="rci-router-trigger-2">Router 2</button>
+    <button type="button" id="rci-router-trigger-checkout">Router Checkout</button>
+
+    <hr/>
+
+    <br /> <br />
+
+    <h2>Dojo Action Service</h2>
+    <p>A event will fire every time an element with attribute <strong>rci-action-trigger</strong> is clicked.  The attribute should include a text label which will be used to identify the action.  Click one of the buttons below:</p>
+    <button type="button" id="rci-dojo-example-1" rci-action-trigger="button-1">Button 1</button>
+    <button type="button" rci-action-trigger="button-2">Button 2</button>
```

Comment:
INFO - is this missing id field is on purpose?

Reply:
Yes - there is a little bit of code which falls back to an attribute syntax if no id is present.

Reply:
Ok